docs: change kubernetes admonitions to red (#909)
redpanda-data/documentation: modules/terms/partials/topic-partition.adoc

=== topic partition
:term-name: topic partition

> :hover-text: A topic may be partitioned through multiple brokers. A "topic partition" represents this logical separation in Redpanda, which is managed natively by Raft and applies voting mechanisms such as Round Robin. In contrast, other Kafka applications use ZooKeeper to manage their partitions.  

Author: ```suggestion
:hover-text: A topic may be partitioned through multiple brokers. A "topic partition" represents this logical separation in Redpanda, which is managed natively by Raft.
```

> :hover-text: A topic may be partitioned through multiple brokers. A "topic partition" represents this logical separation on Redpanda. It is managed by Raft, applying voting mechanisms such as Round Robin. In contrast, other Kafka applications use ZooKeeper to manage their partitions. In Redpanda this is managed natively by Raft. 

Reviewer: ```suggestion
:hover-text: A topic may be partitioned through multiple brokers. A "topic partition" represents this logical separation in Redpanda, which is managed natively by Raft and applies voting mechanisms such as Round Robin. In contrast, other Kafka applications use ZooKeeper to manage their partitions.  
```

Reviewer 2: @Deflaimun Round Robin should not be capitalized

Reviewer 2: Actually, for hover text, it may be best just to end this with "...natively by Raft." You can put in a line break and put the rest (about Zookeeper) below, so it shows in the glossary but not the hover text.
